### ISSUE TITLE
Please hurry up and fix OMMI Sprint

### ISSUE THREAD

**Issue by Tritiko** [SkidderMC/FDPClient#885]
![image](https://github.com/SkidderMC/FDPClient/assets/113193082/9f3e2091-dbd8-4ecb-9add-e727c9d2a97a)
cancle c0b can bypass some Anti-cheating omni sprint.Not limited to Vulcan.For example, Hypixel Universocraft.It can be used for sprint scaffold and others.Please fix this because he is really important.Also, I know it's caused by noslow.I know there's a vulcan sprint disabler, but this doesn't work for other anti-cheats.

**Comment by C00LC0D35** [comment]
config issue

**Comment by Tritiko** [comment]
> config issue

No, if you turn off Noslow, Cancle C0b can be bypassed because of Noslow's cache packets.The problem stems from the fact that Noslow hasn't written separately to consume bow, if you use the version of Cancle C0B at that time, it's completely bypassed.

**Comment by Tritiko** [comment]
> config issue

 If you turn off Noslow, Cancle C0B can bypass Omni Sprint entirely.This reason comes from noslow's cache package.It leads to the chance that using noslow will make the server detect that you are in the sprint.You can open the Cancle C0B in the test suit, then use the omni sprint, sprint backwards with the sword, and right-click fast. Soon there will be sprint vl. It can be cleared by simply re-entering the world, but vl will appear again when using Noslow. This problem arises when writing consume bow packages separately from noslow. This problem would not have occurred with fdpclient before that.

**Comment by Tritiko** [comment]
> config issue

NO.  If you turn off Noslow, Cancle C0B can bypass Omni Sprint entirely.This reason comes from noslow's cache package.It leads to the chance that using noslow will make the server detect that you are in the sprint.You can open the Cancle C0B in the test suit, then use the omni sprint, sprint backwards with the sword, and right-click fast. Soon there will be sprint vl. It can be cleared by simply re-entering the world, but vl will appear again when using Noslow. This problem arises when writing consume bow packages separately from noslow. This problem would not have occurred with fdpclient before that.

**Comment by Tritiko** [comment]
Plz fix that
